src/App.js

```diff
@@ -1,33 +1,61 @@
 import React from 'react';
-import { Routes, Route } from 'react-router-dom';
-// import { useSelector } from 'react-redux';
+import {
+  Routes, Route, Navigate,
+} from 'react-router-dom';
+import { useSelector } from 'react-redux';
 import Cars from './components/cars/Cars';
 import NavBar from './components/navbar/NavBar';
 import ReserveForm from './components/reservations/ReserveForm';
 import Registration from './components/auth/Registration';
 import Login from './components/auth/Login';
 import MyReservations from './components/reservations/MyReservations';
-import AddCarFrom from './components/reservations/AddCarFrom';
+import AddCarForm from './components/reservations/AddCarForm';
 import DeleteCarForm from './components/reservations/DeleteCarForm';
-
+import CarDetails from './components/cars/CarDetails';
+import MobileNavBar from './components/navbar/MobileNavBar';
+import ProtectedAdminRoutes from './components/auth/ProtectedAdminRoutes';
+import ProtectedRoutes from './components/auth/ProtectedRoutes';
 import './App.css';
+import './media-queries-breakpoints/Small.css';
+import './media-queries-breakpoints/Large.css';
+import './media-queries-breakpoints/XLarge.css';
+import './media-queries-breakpoints/XXLarge.css';
+import './media-queries-breakpoints/Desktop.css';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this one and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/212441636-31e51216-3269-4b51-94e4-e0957b528b8b.png)

Reply:
All functions changed to ES6 arrow function